### DISCUSSION TITLE
model for Japanese

### DISCUSSION THREAD

**Discussion by KeisukeNish** [BenWiseman/sentiment.ai#14] [Q&A] [answered]
Hi, Ben.

I want  to use `sentiment_score`  to a vector with Japanese texts.

In this case, should I use `method = "multi.large"`?

Are there any other options to specify a language other than English?

Thanks.

**Comment by BenWiseman** [comment, marked as answer]
That's correct, currently there are no language-specific models besides English. If, however, you come across one that is open source, we can add it in!

Sometimes it can get better results if you use a translation to English and score the sentiment with en.large - the multi lingual model is using embeddings as a kind of tacit translation, so I've found a direct & dedicated translation first can reduce the error. Unfortunately I don't have large enough labeled datasets in other languages to benchmark that approach, so it's just what I've found anecdotally. 

Google & microsoft have APIs with relatively generous free tiers that's a viable & worthwhile path for you to explore.
